src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -983,15 +991,16 @@ public void SetCookies(Uri uri, string cookieHeader)
     }
 
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     internal struct PathList
     {
         // Usage of PathList depends on it being shallowly immutable;
         // adding any mutable fields to it would result in breaks.
-        private readonly SortedList<string, CookieCollection> m_list; // Do not rename (binary serialization)
+        private readonly SortedList m_list; // Do not rename (binary serialization)
 
-        public static PathList Create() => new PathList(new SortedList<string, CookieCollection>(PathListComparer.StaticInstance));
+        public static PathList Create() => new PathList(SortedList.Synchronized(new SortedList(PathListComparer.StaticInstance)));
```

Comment:
I know. That's why I changed it: https://github.com/dotnet/corefx/pull/20870/files#diff-f48fff7b7909f489c5e8b8599c22843cR614